circle.yml

```diff
@@ -5,9 +5,9 @@ test:
   pre:
     - ? |
           # make the build-server and i18n string data in parallel
-          if [[ "$CIRCLE_NODE_INDEX" == 0 ]]; then NODE_ENV=test make build-server; fi
+          if [[ "$CIRCLE_NODE_INDEX" == 0 ]]; then NODE_ENV=test npm run build-server; fi
```

Comment:
We could, but... why? CircleCI is passing using the new build system, which gives some reassurance that it works.